docs/source/api.rst

```diff
@@ -58,13 +58,15 @@ Traversal
    retworkx.dfs_edges
    retworkx.bfs_successors
    retworkx.bfs_search
+   retworkx.dijkstra_search
```

Comment:
One enhancement we could do on the documentation after #456 and #469 is having a subsection in the  Traversal for the visitors, including Dijkstra's. Their usage is quite different from the other methods, I think they deserve to have its own subsection